util/maven/generate_poms.py

```diff
@@ -58,6 +58,29 @@ def pom_deps(label):
         'name': 'Dagger',
         'artifact': 'dagger',
     },
+    '//gwt:gwt': {
+        'name': 'Dagger GWT',
+        'artifact': 'dagger-gwt',
+        'manual_dependencies': '''
```

Comment:
One last thing (I think!): Let's specify these as their gradle-like strings: `'com.google.dagger:dagger:sources:${project.version}'`. Then these can become a list, and we don't have to dedent or do anything so weird in generate_pom.